File: BackEnd/FavouriteService/src/main/java/com/capstone/entity/FavouriteRestaurant.java
package com.capstone.entity;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Document
public class FavouriteRestaurant {
    @Id
    private int restaurantId;
    private String restaurantName;
    private String location;
    private String restaurantImage;
    private double rating;
    private String emailId;
}

File: FrontEnd/src/app/model/favourite-restaurant.ts
export class FavouriteRestaurant {
    public restaurantId : number;
    public restaurantName : string ;
    public location : string ;
    public restaurantImage : string ;
    public rating : string ;
    public emailId : string;

    constructor(  restaurantId : number, restaurantName : string, location : string, restaurantImage : string ,
     rating : string, emailId : string) {
      this.restaurantId = restaurantId;
      this.restaurantName = restaurantName;
      this.location = location;
      this.restaurantImage = restaurantImage;
      this.rating = rating;
      this.emailId = emailId
    }
}
